sdk/search/search-documents/src/generated/data/models/index.ts

```diff
@@ -174,6 +174,8 @@ export interface SearchRequest {
   semanticFields?: string;
   /** The query parameters for vector and hybrid search queries. */
   vector?: Vector;
```

Comment:
.NET removed this with a README transform like https://github.com/azure/azure-sdk-for-net/blob/main/sdk/search/Azure.Search.Documents/src/autorest.md#remove-vector-property

We can also leave for now this until the swagger gets updated if it's not visible to customers.

Reply:
Fixed in this followup PR #26777